2025 Q4 community blog post (#2474)
writethedocs/www: docs/blog/2025-Q4-community-board.rst

.. post:: Nov 21, 2025
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q4-2025
=======================================================

Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share our third quarterly update for 2025 with you. 

To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

> Q3-2025 quarterly meeting highlights

Author: ```suggestion
Q4-2025 highlights
```

------------------------------------

For this quarter, the board meeting was canceled due to the proximity to the inaugural Berlin conference. This was the first time that the board did not meet in real-time, since some members were traveling to the conference. 

Fear not, the team leads provided their updates asynchronously, so we still have some news for you!

> Here are the highlights from the meeting:

Author: ```suggestion
Fear not, the team leads provided their updates asynchronously, so we still have some news for you!

```

    These discussions focused on how we can expand the global community in the face of the changing landscape for conferences, meetups, and online spaces. The CoreOps team will follow up with specific team leads to discuss ideas related to their teams, and the board will communicate proposals and initiatives to the community as we refine them. 
    
**Slack moderation** 
    Janine Chan reported that there's a small trend where scammers try to recruit job seekers for location-based scams. This is mostly done by DMs, which makes it more difficult for the moderators to catch these. Thankfully, the WTD community is sharp and engaged, so the moderation team was informed directly by community members. 

    We would like to remind the community that Slack interactions are a part of our official spaces, so if anyone receives unusual or suspicious DMs, please share them with the Slack moderators so they can ban any bad actors. 

**WTD events in Africa**

>     The Nigeria meetup held their first summit earlier in November 2025, and <thoughts? recap links?>. In addition, the Kenya meetup organizers are highly engaged and are discussing more expanded community activity. We hope to see more documentarian community activities in this continent, and will update when additional events are announced. 

Reviewer: Kenya had a conference! It isn't a discussion, it already happened earlier this year.

>     The Nigeria meetup held their first summit earlier in November 2025, and <thoughts? recap links?>. In addition, the Kenya meetup organizers are highly engaged and are discussing more expanded community activity. We hope to see more documentarian community activities in this continent, and will update when additional events are announced. 

Reviewer: ```suggestion
    The Nigeria meetup held their first summit earlier in November 2025, and <thoughts? recap links?>. In addition, some of the Kenya conference organizers are highly engaged and are discussing more expanded community activity. We hope to see more documentarian community activities in this continent, and will update when additional events are announced. 
```

>     The Nigeria meetup held their first summit earlier in November 2025, and <thoughts? recap links?>. In addition, some of the Kenya conference organizers are highly engaged and are discussing more expanded community activity. We hope to see more documentarian community activities in this continent, and will update when additional events are announced. 

Author: ```suggestion
    The Nigeria meetup held their first summit earlier in November 2025, and some of the organizers of the Kenya conference (held earlier this year) are highly engaged in discussing more expanded community activity. We hope to see more documentarian community activities in this continent, and will update when additional events are announced. 
```

>     The Nigeria meetup held their first summit earlier in November 2025, and <thoughts? recap links?>. In addition, the Kenya meetup organizers are highly engaged and are discussing more expanded community activity. We hope to see more documentarian community activities in this continent, and will update when additional events are announced. 

Author: @ericholscher by now the summit is done, do we want to shoutout anything or add any links?

Reviewer: No link yet.

**Newsletter team changes**

>     The newsletter team is sad to announce that two of their members will be stepping down from their roles. This is not unexpected since the team includes mostly volunteers, but we will miss Hillary Fraley and Royce Cook. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack if interested. 

Reviewer 2: > This is not unexpected since the team includes mostly volunteers, but we will miss Hillary Fraley and Royce Cook.

I feel this is a disservice to our volunteers, can we rephrase.

Author: @plaindocs I was trying to imply that this is an unforunate but unavoidable aspect because volunteers shouldn't feel like they have to stay if life changes or whatnot. Got any suggestions? cc @CollierCZ

Reviewer 3: Could it be something like this?

```suggestion
    The newsletter team is sad to announce that two of their members will be stepping down from their roles. We understand that volunteers move on when the time is right, but we will miss Hillary Fraley and Royce Cook. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack if interested. 
```

Reviewer 2: Nice!  Works for me

Reviewer: Yea this is better.. but we aren't going to get much distribution on this post, so it should definitely be noted more in the newsletter as a callout.

>     The newsletter team is sad to announce that two of their members will be stepping down from their roles. This is not unexpected since the team includes mostly volunteers, but we will miss <do we want to name names?>. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack or email if interested. 

Reviewer 3: ```suggestion
    The newsletter team is sad to announce that two of their members will be stepping down from their roles. This is not unexpected since the team includes mostly volunteers, but we will miss Hillary Fraley and Royce Cook. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack if interested. 
```

>     The newletter team is sad to announce that two of their members will be stepping down from their roles. This is not unexpected since the team includes mostly volunteers, but we will miss <do we want to name names?>. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack or email if interested. 

Author: @CollierCZ please see question in brackets, how detailed do we want to get with this?

Reviewer 3: I would name names and suggest Slack as the method since they'll have to be in there for the newsletter part anyway.

>     The newletter team is sad to announce that two of their members will be stepping down from their roles. This is not unexpected since the team includes mostly volunteers, but we will miss <do we want to name names?>. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack or email if interested. 

Author: ```suggestion
    The newsletter team is sad to announce that two of their members will be stepping down from their roles. This is not unexpected since the team includes mostly volunteers, but we will miss <do we want to name names?>. If you've been considering joining one of the WTD community teams, the newsletter is a great place to start and stay! Please contact Aaron Collier on Slack or email if interested. 
```
